### ISSUE TITLE
ER: Review the project.yml file template and files to fix the section for SDG info

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#6277]
### Emergent Requirement - Problem
The project.yml file template needs updating to improve SDG formatting and reduce duplicate data.

### Issue you discovered this emergent requirement in
- Issue found in #4187 
- File where problem can be seen https://github.com/hackforla/website/blob/ad2bc55c4cf21e4f3c9abcecb677c08c6389b75f/_projects/lucky-parking.md?plain=1#L80C1-L80C25
- Should not be `# program area card data` instead should be `# SDG info`
- Also, where are we in the process of deployment of SDG to all projects, and in the future making the project.yml file only reference the SDG for the finding of assets needed (logo, color and alt text)

### Date discovered 
2024-01-29

### Did you have to do something temporarily
- [ ] YES
- [ ] NO

### Who was involved
@ExperimentsInHonesty @JessicaLucindaCheng 

### What happens if this is not addressed
redundant code, etc.

### Recommended Action Items
- [x] Make a new issue
- [ ] Discuss with team
- [ ] Let a Team Lead know

### Potential solutions [draft] 
- Write issue(s) for `# program area card data` to become `# SDG info` in _project Markdown files

**Comment by JessicaLucindaCheng** [comment]
Issue Writing
- Progress: Has been added to #4153 as an Action Item. I will need to get some of the other issues completed and my blocker answered before we can refactor out all the redundant code in the project Markdown files.
- Blockers: These questions need to be answered: https://github.com/hackforla/website/issues/4153#issuecomment-1925844555
- Availability: 6 hours
- ETA: For writing issue(s) for changing `# program area card data` to become `# SDG info` in _project Markdown files, Monday, February 19, 2024.

### Question
> Also, where are we in the process of deployment of SDG to all projects, and in the future making the project.yml file only reference the SDG for the finding of assets needed (logo, color and alt text)?

  - **Answer:** I need to finish writing issues for adding SDG info files, logo, color scss, etc, but I need these questions answered in order to finish writing those issues: https://github.com/hackforla/website/issues/4153#issuecomment-1925844555

**Comment by JessicaLucindaCheng** [comment]
Issue Writing
- Progress: Has been added to #4153 as an Action Item. I will need to get some of the other issues completed and my blocker answered before we can refactor out all the redundant code in the project Markdown files.
- Blockers: These questions need to be answered: https://github.com/hackforla/website/issues/4153#issuecomment-1925844555
- Availability: 6 hours
- ETA: For writing issue(s) for changing `# program area card data` to become `# SDG info` in _project Markdown files, Monday, March 11, 2024.

### Question
> Also, where are we in the process of deployment of SDG to all projects, and in the future making the project.yml file only reference the SDG for the finding of assets needed (logo, color and alt text)?

  - **Answer:** I need to finish writing issues for adding SDG info files, logo, color scss, etc, but I need these questions answered in order to finish writing those issues: https://github.com/hackforla/website/issues/4153#issuecomment-1925844555

**Comment by JessicaLucindaCheng** [comment]
Issue Writing
- Progress: Has been added to #4153 as an Action Item. I will need to get some of the other issues completed and my blocker answered before we can refactor out all the redundant code in the project Markdown files.
- Blockers: Water-damaged laptop
- Availability: TBD - Need to resolve my computer problems
- ETA: TBD; dependent on my fixing my computer

### Notes
- See https://github.com/hackforla/website/issues/4153#issuecomment-1925844555

### Question
> Also, where are we in the process of deployment of SDG to all projects, and in the future making the project.yml file only reference the SDG for the finding of assets needed (logo, color and alt text)?

  - **Answer:** I need to finish writing issues for adding SDG info files, logo, color scss, etc, but I need these questions answered in order to finish writing those issues: https://github.com/hackforla/website/issues/4153#issuecomment-1925844555

**Comment by JessicaLucindaCheng** [comment]
Issue Writing
- Progress: Has been added to #4153 as an Action Item. I will need to get some of the other issues completed and my blocker answered before we can refactor out all the redundant code in the project Markdown files.
- Blockers: Water-damaged laptop but it works now
- Availability: TBD - I need this week to catch up and prioritize what needs to be done.
- ETA: TBD; I need this week to catch up and prioritize what needs to be done.

### Notes
- See https://github.com/hackforla/website/issues/4153#issuecomment-1925844555

### Question
> Also, where are we in the process of deployment of SDG to all projects, and in the future making the project.yml file only reference the SDG for the finding of assets needed (logo, color and alt text)?

  - **Answer:** I need to finish writing issues for adding SDG info files, logo, color scss, etc, but I need these questions answered in order to finish writing those issues: https://github.com/hackforla/website/issues/4153#issuecomment-1925844555

**Comment by JessicaLucindaCheng** [comment]
Issue Writing
- Progress: Has been added to #4153 as an Action Item. I will need to get some of the other issues completed.
- Blockers: None
- Availability: TBD
- ETA: TBD

### Notes
- See https://github.com/hackforla/website/issues/4153#issuecomment-1925844555

### Question
> Also, where are we in the process of deployment of SDG to all projects, and in the future making the project.yml file only reference the SDG for the finding of assets needed (logo, color and alt text)?

  - **Answer:** I need to finish writing issues for adding SDG info files, logo, color scss, etc, but I need these questions answered in order to finish writing those issues: https://github.com/hackforla/website/issues/4153#issuecomment-1925844555

**Comment by JessicaLucindaCheng** [comment]
Issue Writing
- Progress: Has been added to #4153 as an Action Item. I will need to get some of the other issues completed.
- Blockers: None
- Availability: TBD
- ETA: TBD

### Notes
- See https://github.com/hackforla/website/issues/4153#issuecomment-1925844555

### Question
> Also, where are we in the process of deployment of SDG to all projects, and in the future making the project.yml file only reference the SDG for the finding of assets needed (logo, color and alt text)?

  - **Answer:** I need to finish writing issues for adding SDG info files, logo, color scss, etc, but I need these questions answered in order to finish writing those issues: https://github.com/hackforla/website/issues/4153#issuecomment-1925844555

**Comment by JessicaLucindaCheng** [comment]
Issue Writing
- Progress: Has been added to #4153 as an Action Item. I will need to get some of the other issues completed.
- Blockers: None
- Availability: TBD
- ETA: TBD

### Notes
- See https://github.com/hackforla/website/issues/4153#issuecomment-1925844555

### Question
> Also, where are we in the process of deployment of SDG to all projects, and in the future making the project.yml file only reference the SDG for the finding of assets needed (logo, color and alt text)?

  - **Answer:** I need to finish writing issues for adding SDG info files, logo, color scss, etc, but I need these questions answered in order to finish writing those issues: https://github.com/hackforla/website/issues/4153#issuecomment-1925844555

**Comment by JessicaLucindaCheng** [comment]
Issue Writing
- Progress: Has been added to #4153 as an Action Item. I will need to get some of the other issues completed. Currently working on #4153 
- Blockers: None
- Availability: TBD
- ETA: TBD

### Notes
- See https://github.com/hackforla/website/issues/4153#issuecomment-1925844555

### Question
> Also, where are we in the process of deployment of SDG to all projects, and in the future making the project.yml file only reference the SDG for the finding of assets needed (logo, color and alt text)?

  - **Answer:** I need to finish writing issues for adding SDG info files, logo, color scss, etc, but I need these questions answered in order to finish writing those issues: https://github.com/hackforla/website/issues/4153#issuecomment-1925844555
